Entities/Common/Building/PlacementCommon.as

```diff
@@ -113,10 +113,10 @@ bool isBuildableAtPos(CBlob@ this, Vec2f p, TileType buildTile, CBlob @blob, boo
 	if (
 		(!canPlaceOnBackground || !map.isTileBackgroundNonEmpty(backtile)) &&      // can put against background
 		!(                                              // can put sticking next to something
-			canPlaceNextTo(map, left) || (canPlaceOnBackground && map.isTileBackgroundNonEmpty(left))  ||
-			canPlaceNextTo(map, right) || (canPlaceOnBackground && map.isTileBackgroundNonEmpty(right)) ||
-			canPlaceNextTo(map, up)   || (canPlaceOnBackground && map.isTileBackgroundNonEmpty(up))    ||
-			canPlaceNextTo(map, down) || (canPlaceOnBackground && map.isTileBackgroundNonEmpty(down))
+			canPlaceNextTo(map, left) || (canPlaceOnBackground && map.isTileBackgroundNonEmpty(left) && !map.isTileGrass(left.type))  ||
```

Comment:
Unsure if hardcoding grass there is the best solution, maybe checking against `isTileSupport` is better?

Reply:
Please take a look at what I changed to see if it is any better. I don't think `canPlaceOnBackground` is needed and it was causing the issue so I removed it. From testing, it didn't seem to change anything with placing spikes and other blob tiles.